inc/admin/customfonts/namespace.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * @author Steel Wagstaff
+ * @license  GPLv3 (or any later version)
+ */
+
+namespace Pressbooks\Admin\CustomFonts;
```

Comment:
I'm not super experienced with creating namespaces, so I might be doing this wrong, but I'm trying new things!